### PR TITLE
KOGITO-5264: Create CLI tool to generate forms from tasks json schemas in Kogito Projects

### PR DESCRIPTION
**Pull request by pefernan**
@tiagobento another WIP PR!

### CODE REVIEW

**Comment by tiagobento** [comment]
@pefernan Build is still failing. I think we should build for x86 on `macOS`. For `macOS` arm64 computers we would rely on Rosetta 2 for now.

**Comment by ederign** [comment]
As alternative, I can manually try with arm if necessary.

**Review by tiagobento — APPROVED** [review]

File: packages/form-generation-tool/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "build:prod": "yarn lint && yarn test && rimraf dist && webpack && run-script-os"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kiegroup/kogito-tooling.git"
+  },
+  "dependencies": {
+    "@kogito-tooling/uniforms-patternfly-codegen": "0.10.0",
+    "uniforms": "3.0.0",
+    "uniforms-bridge-json-schema": "3.0.0",
```
Comment (ljmotta) [inline packages/form-generation-tool/package.json]: Hi @pefernan , could you use the uniforms version `^3.5.1`?  To be the same as the `form` and `uniforms-patternfly-codegen` packages.

**Review by ljmotta — COMMENTED** [review]
Hi @pefernan , I've just made a inline comment about the uniforms version

**Comment by pefernan** [comment]
jenkins retest this please

**Comment by pefernan** [comment]
@ljmotta updated!

**Review by ljmotta — APPROVED** [review]
@pefernan LGTM!

**Review by tiagobento — APPROVED** [review]

**Pull request merged**